Assets/Vimeo/Scripts/Player/VimeoPlayer.cs

```diff
@@ -116,6 +118,7 @@ private void SetupVideoController()
                     controller.OnPlay += VideoPlay;
                     controller.OnPause += VideoPaused;
                     controller.OnFrameReady += VideoFrameReady;
+                    controller.OnPlayLogging += LogPlay;
```

Comment:
so does this mean play logging will be default on for everyone?

Reply:
Right now it is. Do you think the controller should have a second state boolean like `controller.playLoggingEnabled`?